server/server.js

```diff
@@ -1,15 +1,135 @@
+const dotenv = require("dotenv");
+dotenv.config();
+
 const express = require("express");
+const cors = require("cors");
+const bodyParser = require("body-parser");
+const PORT = process.env.PORT || 5500;
 const app = express();
-const port = process.env.PORT || 5000;
+const { Pool } = require("pg");
+app.use(cors());
+app.use(bodyParser.json());
+
+const videosPool = new Pool({
+  host: process.env.DB_HOST,
+  port: process.env.DB_PORT,
+  database: process.env.DB_NAME,
+  user: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+});
+
+app.get("/videos", async (req, res) => {
+  const sortType = req.query.sort;
+  try {
+    let query = "SELECT * FROM videos";
+    if (sortType !== "asc") {
+      query += " ORDER BY ratings DESC";
+    } else {
+      query += " ORDER BY ratings";
+    }
+    const allVideos = await videosPool.query(query);
+    res.json(allVideos.rows);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error.message);
+  }
+});
+
+app.post("/videos/:id/like", async (req, res) => {
+  const videoId = req.body.id;
```

Comment:
You can also get it from `req.params.id` so not essential to send it in the body

Reply:
Yes, it is true, will make the necessary changes. thank you very much. 😺